modules/openxr/editor/openxr_editor_plugin.h

```diff
@@ -32,13 +32,15 @@
 #define OPENXR_EDITOR_PLUGIN_H
 
 #include "openxr_action_map_editor.h"
+#include "openxr_select_runtime.h"
 
 #include "editor/editor_plugin.h"
 
 class OpenXREditorPlugin : public EditorPlugin {
 	GDCLASS(OpenXREditorPlugin, EditorPlugin);
 
 	OpenXRActionMapEditor *action_map_editor = nullptr;
+	OpenXRSelectRuntime *select_runtime = nullptr;
```

Comment:
Can you add a `#ifndef ANDROID_ENABLED` so this is disabled for the Android editor.

Reply:
That makes sense :)